src/mono/mono/mini/jit.h

```diff
@@ -58,8 +58,11 @@ typedef enum {
 	MONO_AOT_MODE_FULL,
 	/* Same as full, but use only llvm compiled code */
 	MONO_AOT_MODE_LLVMONLY,
-	/* Uses Interpreter, JIT is disabled and not allowed,
-	 * equivalent to "--full-aot --interpreter" */
+	/*
+	 * Uses Interpreter, JIT is disabled and not allowed,
+	 * equivalent to "--full-aot --interpreter".
```

Comment:
```suggestion
	 * equivalent to "--full-aot-interp".
```

Not sure if it's equivalent to both, but we do have a dedicated option for this mode, so it's probably less confusing to just say so.